.github/workflows/coverage_report.yaml

```diff
@@ -0,0 +1,34 @@
+name: coverage_report
+
+on:
+  schedule:
```

Comment:
Better to add a `workflow_dispatch` so we can generate the report on demand.

Reply:
added